### PR TITLE
Add change log of v0.7.0

### PR DESCRIPTION
**Pull request by Yshuo-Li**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+## v0.7.0 (30/04/2021).
+
+**Highlights**
+
+1. Support BasicVSR
+2. Support IconVSR
+3. Support RDN
```
Comment (innerlee) [inline docs/changelog.md:9]: Add conference info such as

3. Support RDN (CVPR'2018)

File: docs/changelog.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+- Fix link in README ([#256](https://github.com/open-mmlab/mmediting/pull/256))
+- Tiny fix of `mmedit/apis/test.py` ([#261](https://github.com/open-mmlab/mmediting/pull/261))
+- Tiny Fix of restoration_inference ([#270](https://github.com/open-mmlab/mmediting/pull/270))
+- Fix usage of channel_order in loading.py ([#271](https://github.com/open-mmlab/mmediting/pull/271))
+- Fix the command of inpainting ([#278](https://github.com/open-mmlab/mmediting/pull/278))
+- Fix `preprocess_vimeo90k_dataset.py` args name ([#281](https://github.com/open-mmlab/mmediting/pull/281))
+
+**Breaking Changes**
+
+- Add zh-CN README ([#262](https://github.com/open-mmlab/mmediting/pull/262))
```
Comment (innerlee) [inline docs/changelog.md:35]: Readme couldn't break any code, right?

File: docs/changelog.md [hunk 3]
```diff
@@ -1,5 +1,48 @@
...
+
+**New Features**
+
+- Add RDN ([#233](https://github.com/open-mmlab/mmediting/pull/233), [#260](https://github.com/open-mmlab/mmediting/pull/260), [#280](https://github.com/open-mmlab/mmediting/pull/280))
+- Add MultipleGT datasets ([#238](https://github.com/open-mmlab/mmediting/pull/238))
+- Add BasicVSR and IconVSR ([#245](https://github.com/open-mmlab/mmediting/pull/245), [#252](https://github.com/open-mmlab/mmediting/pull/252), [#253](https://github.com/open-mmlab/mmediting/pull/253), [#254](https://github.com/open-mmlab/mmediting/pull/254), [#264](https://github.com/open-mmlab/mmediting/pull/264), [#274](https://github.com/open-mmlab/mmediting/pull/274))
+- Add IconVSR([#252](https://github.com/open-mmlab/mmediting/pull/252), [#264](https://github.com/open-mmlab/mmediting/pull/264))
+- Add 'GenerateFrameIndicesForRecurrent' ([#258](https://github.com/open-mmlab/mmediting/pull/258))
+- Add zh-CN README ([#262](https://github.com/open-mmlab/mmediting/pull/262))
+- Add onnx evaluation tool ([#279](https://github.com/open-mmlab/mmediting/pull/279))
```
Comment (innerlee) [inline docs/changelog.md:19]: Put this item to highlights

File: docs/changelog.md [hunk 4]
```diff
@@ -1,5 +1,48 @@
...
+3. Support RDN
+
+**New Features**
+
+- Add RDN ([#233](https://github.com/open-mmlab/mmediting/pull/233), [#260](https://github.com/open-mmlab/mmediting/pull/260), [#280](https://github.com/open-mmlab/mmediting/pull/280))
+- Add MultipleGT datasets ([#238](https://github.com/open-mmlab/mmediting/pull/238))
+- Add BasicVSR and IconVSR ([#245](https://github.com/open-mmlab/mmediting/pull/245), [#252](https://github.com/open-mmlab/mmediting/pull/252), [#253](https://github.com/open-mmlab/mmediting/pull/253), [#254](https://github.com/open-mmlab/mmediting/pull/254), [#264](https://github.com/open-mmlab/mmediting/pull/264), [#274](https://github.com/open-mmlab/mmediting/pull/274))
+- Add IconVSR([#252](https://github.com/open-mmlab/mmediting/pull/252), [#264](https://github.com/open-mmlab/mmediting/pull/264))
+- Add 'GenerateFrameIndicesForRecurrent' ([#258](https://github.com/open-mmlab/mmediting/pull/258))
+- Add zh-CN README ([#262](https://github.com/open-mmlab/mmediting/pull/262))
```
Comment (innerlee) [inline docs/changelog.md:18]: All doc-related belongs to improvements section

File: docs/changelog.md (hunk 2 above)
Comment (Yshuo-Li) [reply docs/changelog.md:35]: Yes

**Review by innerlee — APPROVED** [review]

**Pull request merged**